index.html

```diff
@@ -370,20 +382,24 @@ <h3>Use Cases and Requirements</h3>
         </li>
         <li>
 <a>Holders</a> are be able to freely choose and change the agents they employ
-to help them manage and share their <a>verifiable credentials</a>.
+to help them manage their <a>verifiable credentials</a> and generate and share <a>verifiable presentations</a>.
         </li>
         <li>
-<a>Holders</a> that share <a>verifiable credentials</a> are not required to
+<a>Holders</a> that share <a>verifiable presentations</a> never need to
 reveal the identity of the <a>verifier</a> to <a>issuers</a>.
         </li>
         <li>
-A <a>verifiable credential</a> is expressed in one or more standard,
+<a>Holders</a> in <a>presentations</a> can either disclose the attributes or satisfy <a>predicates</a>
```

Comment:
"in presentations" is odd language. Perhaps "can use presentations" instead?

Reply:
Changed the language.